content/SCALE/SystemSettings/ScaleShell.md

```diff
@@ -4,42 +4,45 @@ geekdocCollapseSection: true
 weight: 60
 ---
 
-The SCALE shell is a convenient means to run command lines tools, configure different system settings, or find log files or debug information.
-The shell opens with the root user logged in.
+The SCALE **Shell** is convenient for running command lines tools, configuring different system settings, or finding log files and debug information.
+The **Shell** opens with the root user logged in.
+
+{{< hint danger >}}
+**Warning: the supported mechanisms for making configuration changes are the TrueNAS WebUI, CLI, and API exclusively. ALL OTHERS ARE NOT SUPPORTED AND WILL RESULT IN UNDEFINED BEHAVIOR AND MAY RESULT IN SYSTEM FAILURE.**
+{{< /hint >}}
 
 ![Shell](/images/SCALE/shell.png "SCALE Shell")
 
-The *Set font size slider* adjusts the size of text displayed in the shell.
-Click *RESTORE DEFAULT* to reset the shell font and size.
+The *Set font size slider* adjusts the **Shell**'s displayed text size.
+Click *RESTORE DEFAULT* to reset the **Shell** font size.
 
-Shell command history is available for the current session.
+The **Shell** stores the command history for the current session.
 Use the <kbd>Up</kbd> and <kbd>Down</kbd> arrow keys to scroll through previously entered commands.
 Edit the command if desired, then press <kbd>Enter</kbd> to re-enter the command.
-Browsing away from the **Shell** screen clears the command history.
+Leaving the **Shell** screen clears the command history.
 
-<kbd>Home</kbd>, <kbd>End</kbd>, and <kbd>Delete</kbd> keys are supported.
-Tab completion is also available.
+The **Shell* supports the <kbd>Home</kbd>, <kbd>End</kbd>, and <kbd>Delete</kbd> keys, as well as <kbd>Tab</kbd> completion.
 Type a few letters and press <kbd>Tab</kbd> to complete a command name or filename in the current directory.
-Right-clicking in the terminal window displays a reminder about using <kbd>Command+c</kbd> and <kbd>Command+v</kbd> or <kbd>Ctrl+Insert</kbd> and <kbd>Shift+Insert</kbd> for copy and paste operations in the shell.
+Right-clicking in the terminal window displays a reminder about using <kbd>Command+c</kbd> and <kbd>Command+v</kbd> or <kbd>Ctrl+Insert</kbd> and <kbd>Shift+Insert</kbd> for copy and paste operations in the **Shell**.
 
 Entering `exit` leaves the session.
 Click *Reconnect* to start a new session.
 
 ## Default Shell
 
-Clicking other web interface menus closes the shell session and stops commands running in the shell.
-[zsh](https://www.zsh.org/) is the default shell, but this is changed by editing the *root* user in **Credentials > Local Users** and choosing a different option in the *Shell* drop down. Most Linux command line utilities are available in the shell.
+Clicking other web interface menus closes the **Shell** session and stops commands running in the **Shell**.
```

Comment:
```suggestion - bold for direct UI mentions not general function or feature, so in both cases this is talking about shell as the screen or is it about the feature/function shell? kinda murky so I suggest this:
Clicking other web interface menus closes the shell* session and stops commands running in the **Shell** screen.
```

Reply:
@micjohnson777 
Is that first `**shell***` supposed to be `**shell?**`

Reply:
yes... dang typo.